editor/components/block-settings-menu/index.js

```diff
@@ -99,6 +100,7 @@ export class BlockSettingsMenu extends Component {
 							{ count === 1 && <BlockHTMLConvertButton uid={ firstBlockUID } role="menuitem" /> }
 							<BlockDuplicateButton uids={ uids } rootUID={ rootUID } role="menuitem" />
 							{ count === 1 && <SharedBlockConvertButton uid={ firstBlockUID } onToggle={ onClose } itemsRole="menuitem" /> }
+							<PluginBlockSettingsMenuGroup.Slot fillProps={ { uids, onClose } } />
```

Comment:
See also related discussion related to using slots in here:
https://github.com/WordPress/gutenberg/pull/7199#issuecomment-396258372

There are a few important point discussed and you can also learn why:
```jsx
<_BlockSettingsMenuFirstItem.Slot fillProps={ { onClose } } />
```
exists in the first place and why it is considered as internal implementation.

Reply:
01b045b takes this and https://github.com/WordPress/gutenberg/pull/7895#discussion_r202657891 into account and exposes the API as `PluginBlockSettingsMenu`.